### DISCUSSION TITLE
How to link / refer to a GitHub project item from another? (Just as one can refer to PRs and issues with #)

### DISCUSSION THREAD

**Discussion by toraritte** [community/community#37721] [Other Feature Feedback, Questions, & Ideas]
I couldn't find the answer in [the docs](https://docs.github.com/en/issues/planning-and-tracking-with-projects), but it is possible that I overlooked it - or missing something totally obvious...

**Comment by toraritte** [comment]
Alright, I think I did miss the basics that up until project items are not converted to issues in a specific repo, they are little more than draft notes, so there is nothing to refer to. Once they are issues (or even PRs?), then business is as usual.

**Comment by ossanna16** [comment]
Hi there @toraritte and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
